### PR TITLE
Update 2020_02_28_Proposal_Cjdns_using_Wireguard_for_encryption.md

### PR DESCRIPTION
**Pull request by Ivansusi021**

### CODE REVIEW

**Comment by benhylau** [comment]
@Ivansusi021 @cjdelisle this patch https://github.com/cjdelisle/cjdns/pull/1227 and the other recent Rust patches to the codebase are associated with this Milestone right?

**Review by cjdelisle — APPROVED** [review]

**Review by backupbrain — APPROVED** [review]
I keep hearing great things about this Rust. Nicely done.

**Review by benhylau — APPROVED** [review]

**Review by neilalexander — APPROVED** [review]

File: projects/2020_02_28_Proposal_Cjdns_using_Wireguard_for_encryption.md [hunk 1]
```diff
@@ -134,3 +134,13 @@ First payment of 8mn PKT made in 16 transactions:
...
 [16](https://pkt-insight.cjdns.fr/#/PKT/pkt/tx/7c8e437d82c222e580838bcc7e0ba521c019bb56c60e2cba816a1b576bdbf76a)
+
+### Milestone 1  2020-12-17
+-----------
+
+* CJDNS build allows Rust code to be compiled along with the C code
+* Interoperability with C code is implemented: certain important C functions are made available for call from Rust and vice versa
+* Key concepts are implemented in Rust: Interface and Message
+* Unit tests are written to show that C-originated Message can be handled in Rust Interface
+* AndroidWrapper is rewritten in Rust: a small Inetrface that demonstrates how everything works together
```
Comment (Arceliar) [inline projects/2020_02_28_Proposal_Cjdns_using_Wireguard_for_encryption.md:145]: Minor typo (I assume): `Inetrface` -> `Interface`

**Review by Arceliar — APPROVED** [review]
I haven't tested it myself (yet), but the milestone itself looks good to me. I noticed a minor typo that I'd ideally like to see fixed, but it's not worth holding things up over.

**Pull request merged**